extension/chrome/settings/modules/auth_denied.htm

```diff
@@ -34,19 +34,12 @@ <h1>Explaining FlowCrypt webmail permissions</h1>
 
     <div class="line">Please write human@flowcrypt.com if you have any questions or concerns.</div>
 
-    <div style="display: none;" class="display_if_rendered_as_subpage">
-      <div class="line">&nbsp;</div>
-      <div class="line">
-        <button class="button green action_auth_proceed">PROCEED</button>&nbsp;&nbsp;<button class="button gray close_page">CLOSE</button>
-      </div>
-    </div>
+    <div class="line">Would you like to proceed to the permissions screen again?</div>
   </div>
 
   <script src="/lib/purify.js"></script>
```

Comment:
I think all scripts can go. In fact the whole file can just become a few lines of js.

Reply:
But it's ok to leave it as html, but in that case you don't need the HEAD, BODY tags etc. Just a bunch of content.

Reply:
But there's `<a href="#" data-swal-page="/chrome/texts/email_provider_auth_security.htm">`, that's why I left `sweetalert2.js`. And it obviously depends on `jquery.min.js`. The only thing is `purify.js`, but I thought it's also needed because goes first.

Reply:
Try removing all of it. It should continue working. I have added this functionality into the methods that render the modal.

![image](https://user-images.githubusercontent.com/6306961/97069895-06b0fb00-15c3-11eb-8edf-369bbc9ddd21.png)

Reply:
Yes, you are right. It works without the scripts. Removed.